onadata/libs/utils/common_tags.py

```diff
@@ -133,3 +133,4 @@
 EXCEL_TRUE = 1
 EXCEL_FALSE = 0
 MEMBERS = 'members'
+XLS_DATE_FIELDS = ['date', 'today']
```

Comment:
I think `datetime` is not being considered, does it not affect that type of field? The other fields to look at are `start` and `end` fields.

Reply:
Seems I missed https://github.com/onaio/onadata/pull/1581#discussion_r262221961.